### PR TITLE
Headless Argument

### PR DESCRIPTION
**Pull request by GitPushPullLegs**
I had to tweak it like this to get headless to work for me, figured you might like it too.

### CODE REVIEW

**Comment by GitPushPullLegs** [comment]
@nateshmbhat just a reminder of this PR. If you'd like to me to tweak it please let me know.

**Comment by nateshmbhat** [comment]
See this answer : https://stackoverflow.com/a/46929945/6665568

That argument is not needed.

**Comment by nateshmbhat** [comment]
What u can do is, not remove that headless parameter line.

File: webbot/webbot.py [hunk 1]
```diff
@@ -52,7 +52,7 @@ def __init__(self, showWindow=True, proxy=None , downloadPath:str=None):
             options.add_argument("--proxy-server={}".format(proxy))
 
         if not showWindow:
-            options.headless = True
```
Comment (nateshmbhat) [inline webbot/webbot.py:55]: Don't remove this line. Let it be.

**Review by nateshmbhat — CHANGES_REQUESTED** [review]

File: webbot/webbot.py (hunk 1 above)
Comment (GitPushPullLegs) [reply webbot/webbot.py:55]: Okay, I've added it back in.

**Review by nateshmbhat — APPROVED** [review]

**Pull request merged**